LatticeCoordinate[2] positions
---
LatticeCoordinate[4] path
